evaluators/embedding/L3NormEvaluator/__init__.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+
+from jina.executors.evaluators.embedding import BaseEmbeddingEvaluator, expand_vector
+
+
+class L3NormEvaluator(BaseEmbeddingEvaluator):
```

Comment:
Let's have it parametrizable by p.

Reply:
This L3norm evaluator could be replaced by the more general Hamming-distance-evaluator